src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -401,9 +402,19 @@ void damped_harmonic_h(
   for (size_t a = 0; a < SpatialDim + 1; ++a) {
     gauge_h->get(a) = (1. - roll_on_h_init) * gauge_h_init.get(a) +
                       get(h_prefac1) * spacetime_unit_normal_one_form.get(a);
+    // In case we want SpEC-style roll-on of gauge
+    if (UNLIKELY(use_spec_style_rollon)) {
+      gauge_h->get(a) += (roll_on_h_init - 1.) * get(h_prefac1) *
+                         spacetime_unit_normal_one_form.get(a);
+    }
```

Comment:
I believe this actually is the intended behavior - the roll-**on** in spec has two gaussian roll-on functions being multiplied. See this [discussion](https://github.com/sxs-collaboration/spectre/pull/1172#discussion_r282175248) with @markscheel, and see `W` and `coefL, coefS` [here](https://github.com/sxs-collaboration/spec/blob/master/EvolutionSystems/GeneralizedHarmonic/DampedHarmonicGaugeItems.cpp#L2596) and [here](https://github.com/sxs-collaboration/spec/blob/master/EvolutionSystems/GeneralizedHarmonic/DampedHarmonicGaugeItems.cpp#L2579) which have the roll-on and roll-off contained in them, respectively, and then both get multiplied later on.

Reply:
You are right. I think this is the behaviour in SpEC since 
commit d6ce9531ffca9f2e600882173441d964fdb03e32 .
Which is after the paper (arXiv:0909.3557) we were thinking of. Does someone know if this change is documented in some later paper?